src/web5.js

```diff
@@ -71,18 +71,19 @@ class Web5 extends EventTarget {
       if (dwnNodes) {
         return this.#send(dwnNodes, { author, data, message, target });
       }
-      return { status: { code: 422, detail: 'No DWN endpoints present in DID document. Request cannot be sent.' } };
+      return { status: { code: 400, detail: 'No DWN endpoints present in DID document. Request cannot be sent.' } };
```

Comment:
why did this change?  "Unprocessable Content" seems like a reasonable response for this, no?

Reply:
Agreed.  Reverted back to `422`